specs/ios_metadata.rb

```diff
@@ -37,7 +37,7 @@ def test_get_window_size
     height = 100
     width = 100
     window_size = { 'height' => height, 'width' => width }
-    @mock_webdriver.expect(:get_window_size, window_size)
```

Comment:
Shouldnt we ideally support both ?

Reply:
there is no function in ruby called `get_window_size` docs:- https://appium.readthedocs.io/en/stable/en/commands/web/window/get-window-size/.
it is a function in python. Maybe when we converted it form python to ruby we didn't test this.
Even our example project has device that is present in device.json file.

Reply:
yes.

Reply:
Can you revert that then ?

Reply:
appium lib > 11.2 is a fix for another customer issue